conda_build/utils.py

```diff
@@ -2103,7 +2109,10 @@ def compute_content_hash(
 def write_bat_activation_text(file_handle, m):
     from .os_utils.external import find_executable
 
-    file_handle.write(f'call "{root_script_dir}\\..\\condabin\\conda_hook.bat"\n')
+    file_handle.write(f'call "{context.root_prefix}\\condabin\\conda_hook.bat"\n')
+    for key, value in context.conda_exe_vars_dict.items():
+        file_handle.write(f'set "{key}={value or ""}"\n')
+    file_handle.write("set CONDA_EXE\n")
```

Comment:
Is this supposed to unset `CONDA_EXE`? If so I think this idiom is more common:

```suggestion
    file_handle.write('set "CONDA_EXE="\n')
```

Reply:
I expect it was working because that particular like failed but `.bat` simply continues executing the rest of the file

Reply:
```suggestion
```